test_windows_full in the CI is broken: pickling difference in stdlibPatches

It appears to have started with https://github.com/lampepfl/dotty/commit/96ced0232b8b91e21127674215ad5ee63dfc33ed, the CI run is https://github.com/lampepfl/dotty/runs/2057136722:
```scala
 class dotty.tools.dotc.reporting.Diagnostic$Error at ?: pickling difference for module class language$ in library\src\scala\runtime\stdLibPatches\language.scala, for details:
Fatal compiler crash when compiling: library\src\scala\runtime\stdLibPatches:

  diff before-pickling.txt after-pickling.txt while compiling library\src\scala\runtime\stdLibPatches\language.scala, library\src\scala\runtime\stdLibPatches\Predef.scala
pickling difference for module class language$ in library\src\scala\runtime\stdLibPatches\language.scala, for details:

  diff before-pickling.txt after-pickling.txt
dotty.tools.dotc.report$.error(report.scala:71)
dotty.tools.dotc.transform.Pickler.testSame(Pickler.scala:145)
dotty.tools.dotc.transform.Pickler.testUnpickler$$anonfun$2(Pickler.scala:135)
scala.runtime.function.JProcedure1.apply(JProcedure1.java:15)
scala.runtime.function.JProcedure1.apply(JProcedure1.java:10)
scala.collection.IterableOnceOps.foreach(IterableOnce.scala:563)
scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:561)
scala.collection.AbstractIterable.foreach(Iterable.scala:919)
scala.collection.IterableOps$WithFilter.foreach(Iterable.scala:889)
dotty.tools.dotc.transform.Pickler.testUnpickler(Pickler.scala:136)
dotty.tools.dotc.transform.Pickler.runOn(Pickler.scala:119)
dotty.tools.dotc.Run.runPhases$4$$anonfun$4(Run.scala:205)
scala.runtime.function.JProcedure1.apply(JProcedure1.java:15)
scala.runtime.function.JProcedure1.apply(JProcedure1.java:10)
scala.collection.ArrayOps$.foreach$extension(ArrayOps.scala:1323)
dotty.tools.dotc.Run.runPhases$5(Run.scala:215)
dotty.tools.dotc.Run.compileUnits$$anonfun$1(Run.scala:223)
scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
dotty.tools.dotc.util.Stats$.maybeMonitored(Stats.scala:67)
dotty.tools.dotc.Run.compileUnits(Run.scala:230)
dotty.tools.dotc.Run.compileSources(Run.scala:166)
dotty.tools.dotc.Run.compile(Run.scala:150)
dotty.tools.dotc.Driver.doCompile(Driver.scala:39)
dotty.tools.dotc.Driver.process(Driver.scala:198)
dotty.tools.dotc.Driver.process(Driver.scala:166)
dotty.tools.vulpix.ParallelTesting$Test.compile(ParallelTesting.scala:490)
dotty.tools.vulpix.ParallelTesting$CompilationLogic.compileTestSource$$anonfun$2$$anonfun$1(ParallelTesting.scala:219)
scala.collection.immutable.List.map(List.scala:246)
dotty.tools.vulpix.ParallelTesting$CompilationLogic.compileTestSource$$anonfun$1(ParallelTesting.scala:219)
scala.util.Try$.apply(Try.scala:210)
dotty.tools.vulpix.ParallelTesting$CompilationLogic.dotty$tools$vulpix$ParallelTesting$CompilationLogic$$compileTestSource(ParallelTesting.scala:220)
dotty.tools.vulpix.ParallelTesting$$anon$2.checkTestSource$$anonfun$1(ParallelTesting.scala:265)
dotty.tools.vulpix.ParallelTesting$Test.tryCompile(ParallelTesting.scala:433)
dotty.tools.vulpix.ParallelTesting$$anon$2.checkTestSource(ParallelTesting.scala:268)
dotty.tools.vulpix.ParallelTesting$Test$LoggedRunnable.run(ParallelTesting.scala:336)
dotty.tools.vulpix.ParallelTesting$Test$LoggedRunnable.run$(ParallelTesting.scala:318)
dotty.tools.vulpix.ParallelTesting$$anon$2.run(ParallelTesting.scala:263)
java.util.concurrent.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1386)
java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:175)
Error:  Test dotty.tools.dotc.BootstrappedOnlyCompilationTests.picklingWithCompiler failed: java.lang.AssertionError: Expected no errors when compiling, failed for the following reason(s):
Error:  
Error:    - encountered 1 test failures(s), took 25.844 sec
Error:      at dotty.tools.vulpix.ParallelTesting$CompilationTest.checkCompile(ParallelTesting.scala:897)
Error:      at dotty.tools.dotc.BootstrappedOnlyCompilationTests.picklingWithCompiler(BootstrappedOnlyCompilationTests.scala:177)
Error:      ...
[info] Test dotty.tools.dotc.BootstrappedOnlyCompilationTests.negWithCompiler started
[info] Test dotty.tools.dotc.BootstrappedOnlyCompilationTests.posMacros started
Foo[T]
[info] Test dotty.tools.dotc.BootstrappedOnlyCompilationTests.runWithCompiler started
foo
[info] Test dotty.tools.dotc.BootstrappedOnlyCompilationTests.runBootstrappedOnly started

================================================================================
Test Report
================================================================================

10 suites passed, 1 failed, 11 total
    library\src\scala\runtime\stdLibPatches failed
```

--- a/compiler/test/dotty/tools/dotc/BootstrappedOnlyCompilationTests.scala
+++ b/compiler/test/dotty/tools/dotc/BootstrappedOnlyCompilationTests.scala
@@ -150,7 +150,7 @@ class BootstrappedOnlyCompilationTests {
 
   @Test def picklingWithCompiler: Unit = {
     val jvmBackendFilter = FileFilter.exclude(List("BTypes.scala", "Primitives.scala")) // TODO
-    val runtimeFilter = FileFilter.exclude(List("Tuple.scala")) // TODO
+    val runtimeFilter = FileFilter.exclude(List("Tuple.scala", "stdLibPatches")) // TODO
     implicit val testGroup: TestGroup = TestGroup("testPicklingWithCompiler")
     aggregateTests(
       compileDir("compiler/src/dotty/tools", picklingWithCompilerOptions, recursive = false),

--- a/library/src/scala/runtime/stdLibPatches/language.scala
+++ b/library/src/scala/runtime/stdLibPatches/language.scala
@@ -52,8 +52,6 @@ object language:
     object symbolLiterals
   end deprecated
 
-  object symbolLiterals
-
   /** Where imported, auto-tupling is disabled.
     *
     * '''Why control the feature?''' Auto-tupling can lead to confusing and

